data/dashboard.yml

```
@@ -41,6 +41,17 @@
         - name: Repositories
           url: /repos.html
           description: All the applications and other repositories we use to build GOV.UK.
+    - name: GOV.UK documentation
+      sites:
+        - name: 2nd-line Support Manuals
```

super nit (sorry!): [house style](https://www.gov.uk/guidance/style-guide/a-to-z-of-gov-uk-style#capitalisation) for capitalisation
```suggestion
        - name: 2nd-line Tech Support manuals
```

I've added suggested edits for the other instances below for convenience